tests/Integration/EnqueuedScriptsTest.php

```diff
@@ -68,13 +68,18 @@ protected function tearDown(): void {
 	 * Helper to execute a GraphQL query with the post URL.
 	 */
 	private function query(): string {
+		// Get the appropriate field name based on WPGraphQL version
+		$location_field = defined( 'WPGRAPHQL_VERSION' ) && version_compare( WPGRAPHQL_VERSION, '1.30.0', '>=' )
+		? 'groupLocation'
+		: 'location';
+
 		return '
 			query GetEnqueuedScripts($uri: String!) {
 				templateByUri(uri: $uri) {
 					enqueuedScripts(first: 1000) {
 						nodes {
 							handle
-							location
+							{$location_field}
```

Comment:
This is the _only_ change that should be made to this file. If there's soething else in this file diff vs `develop`, then the implementation is incorrect.

```suggestion
							groupLocation
```